README.md

```diff
@@ -33,29 +52,26 @@ The directory structure of your new project looks like this:
 
 ```
 .
-├── dataset_paths             <- Keep your datasets here
+├── Library submodule         <- SeismicPro, SeismiQB or PetroFlow as a git submodule
+├── datasets                  <- Keep your datasets here
 ├── docker_containers
-│   └── readme.md
-├── extra                     <- extra helper utilities that are not project-specific, ex. cookiecutter template updater
-│   ├── update_cookiecutter_template.sh <- run this script to merge recent changes in cookiecutter template into your project
-│   └── .cookiecutter.json
 ├── .dockerignore
+├── extra                     <- extra helper utilities that are not project-specific, ex. cookiecutter template updater
+│   ├── .cookiecutter.json
+│   ├── src
+│   └── update_cookiecutter_template.sh   <- run this script to merge recent changes in cookiecutter template into your project
+├── .git
+├── .gitattributes
 ├── .github
 │   └── workflows
 │       └── status.yml
-├── .gitattributes
 ├── .gitignore
 ├── notebooks                 <- Development notebooks
-│   └── readme.md
+├── overview                  <- Notebooks with overview of main results
```

Comment:
What is the main difference between "Development notebooks" and "Notebooks with overview"? Could you show examples?

Reply:
Development notebooks are the ones that are created while experimenting with different approaches to solve a problem. " overview of main results" shows notebooks with final models description, consisetent with standards